client/src/features/sessionsV2/SessionView/SessionView.tsx

```diff
@@ -60,17 +56,17 @@ import {
   SessionStatusV2Label,
   SessionStatusV2Title,
 } from "../components/SessionStatus/SessionStatus";
-import sessionsV2Api from "../sessionsV2.api";
-import { SessionEnvironment, SessionLauncher } from "../sessionsV2.types";
-
+import { DEFAULT_URL } from "../session.utils.ts";
+import { SessionLauncher } from "../sessionsV2.types";
 import MembershipGuard from "../../ProjectPageV2/utils/MembershipGuard";
 import {
   useGetResourceClassByIdQuery,
   useGetResourcePoolsQuery,
 } from "../../dataServices/computeResources.api";
 import { useGetProjectsByProjectIdMembersQuery } from "../../projectsV2/api/projectV2.enhanced-api";
-import UpdateSessionLauncherModal from "../UpdateSessionLauncherModal";
+import UpdateSessionLauncherModal from "../components/SessionModals/UpdateSessionLauncherModal.tsx";
```

Comment:
Oh no, not `.tsx` again :scream: 

```suggestion
import UpdateSessionLauncherModal from "../components/SessionModals/UpdateSessionLauncherModal";
```